src/libs/metadata_view.c

```diff
@@ -745,7 +745,11 @@ static void _metadata_view_update_values(dt_lib_module_t *self)
 
       case md_exif_focus_distance:
         (void)g_strlcpy(text, NODATA_STRING, sizeof(text));
-        if(!(isnan(img->exif_focus_distance) || (fpclassify(img->exif_focus_distance) == FP_ZERO) ))
+        if(img->exif_focus_distance == UINT32_MAX)
```

Comment:
Isn't this just catching a corner case issue? We may want to display **infinity** for a focus distance far before that. Not sure what to propose though.

Reply:
> Isn't this just catching a corner case issue? We may want to display **infinity** for a focus distance far before that.

Actually no. Sorry, I should have given a reference to the standard in the initial comment.

Here is a quote from the definition of the corresponding tag in the EXIF standard:
`Note that if the numerator of the recorded value is hex FFFFFFFF, Infinity shall be indicated; and if the numerator is 0, Distance unknown shall be indicated.`

You can make sure of this here:
https://www.cipa.jp/std/documents/e/DC-008-2012_E.pdf
(see page 60 of the PDF which corresponds to page 55 of the page numbering of the document)

Reply:
All clear now, thanks!